core/renderers/geras/path_object.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview TODO
+ * @author fenichel@google.com (Rachel Fenichel)
+ */
+
+'use strict';
+
+goog.provide('Blockly.geras.PathObject');
+
+goog.require('Blockly.blockRendering.IPathObject');
+goog.require('Blockly.utils.dom');
+
+
+/**
+ * An object that handles creating and setting each of the SVG elements
+ * used by the renderer.
+ * @param {!SVGElement} root The root SVG element.
+ * @constructor
+ * @implements {Blockly.blockRendering.IPathObject}
+ * @package
+ */
+Blockly.geras.PathObject = function(root) {
+  this.svgRoot = root;
+
+  // The order of creation for these next three matters, because that
+  // effectively sets their z-indices.
+
+  /**
+   * @type {SVGElement}
+   * @package
```

Comment:
@Override ?

Reply:
path_object doesn't inherit from a single superclass file. Instead we use an interface, and so no need for @override.